### PR TITLE
Update nw watcher

### PR DESCRIPTION
**Pull request by jrspinella**
## Describe your changes

1. Fixed NW not to create but to lookup existing NW.
2. Removed the 'create_network_watcher'
3. Removed 'is_spoke_deployed_to_same_hub_subscription" as not needed
4. Updated examples to have commercial and government folders.
5.  Added README to all examples to show how to deploy the module
6.  Update gitIgnore to allow tfvars in examples

## Issue number

#23 #24 #19 

## Testing

This module has been tested on Azure Commerical and Government

## Checklist before requesting a review
- [X] The pr title can be used to describe what this pr did in `CHANGELOG.md` file
- [X] I have executed pre-commit on my machine
- [X] I have passed pr-check on my machine

Thanks for your cooperation!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,6 @@
 
 This Overlay terraform module deploys a Management Spoke network using the [Microsoft recommended Hub-Spoke network topology](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/hub-spoke) to be used in a [SCCA compliant Management Network](https://registry.terraform.io/modules/azurenoops/overlays-management-spoke/azurerm/latest). Usually, only one hub in each region with multiple spokes and each of them can also be in separate subscriptions.
 
->If you are deploying the spoke VNet in the same Hub Management Network subscription, then make sure you have set the argument `is_spoke_deployed_to_same_hub_subscription = true`. This helps this module to manage the network watcher, flow logs and traffic analytics resources for all the subnets in the Virtual Network. If you are deploying the spoke virtual networks in separate subscriptions, then set this argument to `false`.
-
 This is designed to quickly deploy hub and spoke architecture in the azure and further security hardening would be recommend to add appropriate NSG rules to use this for any production workloads.
```
Comment (sstjean) [inline README.md:7]: Please stop usimg Co-Pilot for documentation.  It's horrible.

File: examples/Government/basic_mgt_spoke/dev.gov.plan
Comment (sstjean) [inline examples/Government/basic_mgt_spoke/dev.gov.plan:1]: Should the plan file be committed to the repo?

File: examples/Commerical/basic_mgt_spoke/parameters.tfvars [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+# Licensed under the MIT License.
+
+###########################
+## Global Configuration  ##
+###########################
+
+# The prefixes to use for all resources in this deployment
+org_name           = "anoa"   # This Prefix will be used on most deployed resources.  10 Characters max.
+deploy_environment = "dev"    # dev | test | prod
+environment        = "usgovernment" # public | usgovernment
```
Comment (sstjean) [inline examples/Commerical/basic_mgt_spoke/parameters.tfvars:11]: This example for Commercial is targeting Gov.

File: examples/Commerical/basic_mgt_spoke/parameters.tfvars [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+# The default region to deploy to
+default_location = "usgovvirginia"
```
Comment (sstjean) [inline examples/Commerical/basic_mgt_spoke/parameters.tfvars:14]: Also targeting Gov region

File: examples/Commerical/basic_mgt_spoke/parameters.tfvars (hunk 2 above)
Comment (jrspinella) [reply examples/Commerical/basic_mgt_spoke/parameters.tfvars:11]: changed

File: README.md (hunk 1 above)
Comment (jrspinella) [reply README.md]: done

Comment (jrspinella) [reply README.md]: Yes

**Review by sstjean — APPROVED** [review]

**Pull request merged**